examples/pytorch/rgcn/experimental/entity_classify_dist.py

```diff
@@ -573,7 +573,8 @@ def main(args):
     if args.num_gpus == -1:
         device = th.device('cpu')
     else:
-        device = th.device('cuda:'+str(args.local_rank))
```

Comment:
Is `args.local_rank` used anywhere?

Reply:
No, but this argument is passed in by `torch.distributed.launch` automatically.

Reply:
Does it mean there is a bug in `torch.distributed.launch`?

Reply:
I think neither. It's just because `g.rank()` does not equal with `args.local_rank` as they are totally irrelevant. DistDGL has its own logic to assign `g.rank()`.